source/utilities.tex

```diff
@@ -114,6 +114,22 @@
   template<class... T>
     using index_sequence_for = make_index_sequence<sizeof...(T)>;
 
+  template<class T> struct tuple_size;
+  template<size_t I, class T> struct tuple_element;
+
+  // \ref{intseq.binding} structured binding support for \tcode{integer_sequence}
```

Comment:
```suggestion
  // \ref{intseq.binding}, structured binding support
```
Our general policy is that there is a comma, and the comments should match the subclause title exactly.

Please fix this in a separate commit (containing a `Fixup` message).